src/main/java/com/example/demo/domain/listentry/ListEntryController.java

```diff
@@ -39,13 +40,20 @@ public ResponseEntity addListEntry(@RequestBody ListEntryDTO listEntry) {
         }
         return ResponseEntity.ok().body(returnedListEntry);
     }
+    @GetMapping("{userID}")
+    @PreAuthorize("hasAuthority('READ_LIST_ENTRY')")
+    public ResponseEntity getAllListEntries(@PathVariable("userID") UUID id) {
+
+        return ResponseEntity.ok().body(listEntryService.getAllListEntries(id));
```

Comment:
error handling if user doesn't exist, send a response to the user that the ID doesn't exist. Similar to other endpoints

Reply:
Preferably use InstanceNotFoundException